merino/curated_recommendations/sections.py

```diff
@@ -38,6 +40,38 @@
 logger = logging.getLogger(__name__)
 
 
+def map_iab_categories_to_section(section_topic: Topic) -> list[str]:
+    """Map an IAB category code to a Section. Source:
+    https://docs.google.com/spreadsheets/d/1R0wzDYgrFkLjo6sxTFvVYThJ4uKz-YVDkYMegbGK9XA/edit?gid=1439764175#gid
+    =1439764175
```

Comment:
I would frame this as mapping a topic to an IAB category.
```suggestion
def map_topic_to_iab_categories(topic: Topic) -> list[str]:
    """Map a topic to an IAB category code. Source:
    https://docs.google.com/spreadsheets/d/1R0wzDYgrFkLjo6sxTFvVYThJ4uKz-YVDkYMegbGK9XA/edit?gid=1439764175#gid
    =1439764175
```